cmd/manager/main.go

```diff
@@ -142,12 +146,22 @@ func main() {
 				os.Exit(1)
 			}
 		} else {
+
 			log.Info("Starting lease controller to report status")
 			leaseUpdater := lease.NewLeaseUpdater(
 				generatedClient,
-				"config-policy-controller",
+				"policy-controller",
```

Comment:
This change was reverted in
- https://github.com/open-cluster-management/config-policy-controller/pull/175

So maybe pull some more PRs from midstream in? I'm not 100% sure of the effect here, but keeping `config-policy-controller` seems clearer to me...

Reply:
Nice catch